client/my-sites/plans/main.jsx

```diff
@@ -99,3 +113,18 @@ module.exports = React.createClass( {
 		);
 	}
 } );
+
+module.exports = connect(
+	( state, props ) => {
```

Comment:
Could we use named functions here? Just to make it easier for us who are starting with redux.